src/api/requestsApi.tsx

```diff
@@ -211,37 +211,48 @@ export const useAddRequest = () => {
   return useMutation(
     ["requests"],
     async (newRequest: RPARequest) => {
-      // We must have the ID of RPADocSet in order to create our DocSet folder
-      let contentTypeId = "";
-      if (contentTypes.data) {
-        contentTypeId =
-          contentTypes.data.find((ct) => ct.Name === "RPADocSet")?.StringId ||
-          "";
+      let id = newRequest.Id ? parseInt(newRequest.Id) : 0;
+      if (id) {
+        await spWebContext.web.lists
+          .getByTitle("requests")
+          .items.getById(id)
+          .update({
```

Comment:
Not a big fan of the useAddRequest hook to be used to update or add.  Maybe split this out as we introduce the Edit capability?

Reply:
I like useAddUpdateRequest better than useMutateRequest, because in my mind you have to have something first, before you can mutate it.  However, RQ uses the term useMutation to cover both creating and modifying, so maybe it would be most appropriate.